packaging/os/apt.py

```diff
@@ -484,6 +493,37 @@ def upgrade(m, mode="yes", force=False, default_release=None,
         m.exit_json(changed=False, msg=out, stdout=out, stderr=err)
     m.exit_json(changed=True, msg=out, stdout=out, stderr=err)
 
+def build_dep(m, pkgspec, cache, force=False,
+              dpkg_options=expand_dpkg_options(DPKG_OPTIONS)):
+    if m.check_mode:
+        check_arg = '--simulate'
+    else:
+        check_arg = ''
+
+    if force:
+        force_yes = '--force-yes'
+    else:
+        force_yes = ''
+
+    packages = ''
+    pkgspec = expand_pkgspec_from_fnmatches(m, pkgspec, cache)
+    for package in pkgspec:
+        name, version = package_split(package)
+        packages += "'%s' " % package
+    if len(packages) == 0:
+        m.exit_json(changed=False)
```

Comment:
With the current parameter setup, using exit_json() in this function would be a bug.  The current parameter setup is that installing build-deps is an optional addition to whatever else we're doing.  So we would have to return to the main function to continue to the next step unless build-dep fails.